ClientSideCode/Carian/screen/drawerScreens/HospitalOverview.js

```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react';
+
+import { StyleSheet, TextInput, View, TouchableOpacity, Text, Alert , Image, ImageBackground,ScrollView, TouchableHighlight } from 'react-native';
+
+import DoctorProfileCard from './Cards/DoctorProfileCard';
+import styles from '../../styles/DoctorProfileStyles';
+import styles1 from '../../styles/commonStyles';
+import HospitalCard from './Cards/HospitalCard';
+import ReviewCard from './Cards/ReviewCard';
+import { SliderBox } from "react-native-image-slider-box";
+import { Table, Row, Rows } from "react-native-table-component";
+import SpecialityCard from './Cards/SpecialityCard';
+
+const image = { uri: "https://thomsonhospitals.com/wp-content/uploads/2019/07/Thomson-Hospital-Kota-Damansara-Specialties-Obstetrics-Gynaecology-Thumbnail.jpg" };
+
+export default class HospitalOverview extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+ 
+      hospitalReviews: [{ id: 0, name: 'Srinivas', rating: 4, date: '11/12/2020', comment: 'Review, criticism imply careful examination of something, formulation of a judgment' },
+      { id: 1, name: 'Nallapati', rating: 4, date: '11/12/2020', comment: 'Review, criticism imply careful examination of something, formulation of a judgment' },
+      { id: 2, name: 'Test', rating: 4, date: '11/12/2020', comment: 'Review, criticism imply careful examination of something, formulation of a judgment' },
+      { id: 3, name: 'Hello', rating: 4, date: '11/12/2020', comment: 'Review, criticism imply careful examination of something, formulation of a judgment' },
+    ],
+
+    hospitalImages: [
+      "https://source.unsplash.com/1024x768/?nature",
+      "https://source.unsplash.com/1024x768/?water",
+      "https://source.unsplash.com/1024x768/?girl",
+      "https://source.unsplash.com/1024x768/?tree"
+
+  ],
+
+  specialistCarddata: [{ image: image, name: 'Family physicians' },
+  { image: image, name: 'Pediatricians' },
+  { image: image, name: 'Geriatric doctors' },
+  { image: image, name: 'Allergists' },
+  { image: image, name: 'Rheumatologists' }],
+
+  headerSlots: ["Days", "24Hours", "Opens At", "Closed at",],
+  workingHours: [
+      ["Mon", "Yes", "-", "-"],
+      ["Tue", "No", "01:00PM", "06:00PM"],
+      ["Wed", "No", "01:00PM", "06:00PM"],
+      ["Thu", "No", "01:00PM", "06:00PM"],
+      ["Fri", "Yes", "-", "-"],
+      ["Sat", "Yes", "-", "-"],
+      ["Sun", "No", "01:00PM", "06:00PM"]],
+ 
+
+      };
+    }
+  
+   
+    render() {
+ 
+      //const name =  this.props.navigation.getParam('name', 'nothing sent')
```

Comment:
create an object in prevoius page and pass it to this page instead on using a big list